labs.yml

```diff
@@ -93,3 +93,11 @@ commands:
       Workflow Group Name\tAccount Group Name
       {{range .}}{{.name_in_workspace}}\t{{.name_in_group}}
       {{end}}
+
+  - name: save-aws-iam-profiles
+    description: '
```

Comment:
Put | then indent the following lines for a proper multiline string literal

Reply:
Done